### ISSUE TITLE
Improve notifications setup

### ISSUE THREAD

**Issue by kristopherjohnson** [kristopherjohnson/MenubarCountdown#15]
Enabling notifications for Menubar Countdown on  Big Sur can be confusing. See https://twitter.com/jleedev/status/1375080727021293568

Not much we can do to work around Apple's requirements, but maybe there is a better way to help the user through initial setup.

**Comment by jleedev** [comment]
Copying images here:

This does not look like it, but it is in fact asking the user for permission:

<img width="370" alt="Screen Shot 2021-03-25 at 9 34 11 AM" src="https://user-images.githubusercontent.com/23022/112489239-a5997900-8d54-11eb-8790-c260ba92c5da.png">

The user must hover over the banner, then choose Options > Allow:

<img width="383" alt="Screen Shot 2021-03-25 at 9 34 07 AM" src="https://user-images.githubusercontent.com/23022/112489307-b3e79500-8d54-11eb-9a64-61c6ada4beee.png">
<img width="483" alt="Screen Shot 2021-03-25 at 9 34 22 AM" src="https://user-images.githubusercontent.com/23022/112489319-b6e28580-8d54-11eb-9cb0-9c5dd371982f.png">

If the user doesn't do this, they must open system preferences and dig through the list of apps in notifications to turn them on.

Knowing nothing about the API or how other apps do this, I have two ideas:

1. When enabling notifications, try to educate the user about what's going to happen.
2. If notifications are denied (and it's possible to interrogate the system about this), offer to open the notifications preference pane.

**Comment by kristopherjohnson** [comment]
Yeah, Apple's API is pretty awful. There may be a way I can customize the text displayed in the permissions request dialog, but there is no way to avoid it entirely or replace it with something in the app.

I think I could check the state of permissions and do something appropriate if the user tries to Start with request for notification but notifications are disabled.

**Comment by kristopherjohnson** [comment]
There is apparently no way to change the text, and workarounds like popping up your own alert first telling the user what to do can get your app rejected: https://stackoverflow.com/questions/41353736/is-it-possible-to-have-a-custom-dialog-message-when-asking-for-notification-perm

So, maybe I need to change the Menubar Countdown "Start" dialog so that it will show the current permission status, and provide a button that opens System Preferences to help the user find the settings?
